lib/sshutils/req.go

```diff
@@ -150,6 +163,12 @@ const (
 	// WindowChangeRequest is a request to change window.
 	WindowChangeRequest = "window-change"
 
+	// FileTransferRequest is a request to approve a file transfer
+	FileTransferRequest = "file-transfer-request"
+
+	// FileTransferRequestResponse is a response to a file transfer request
+	FileTransferRequestResponse = "file-transfer-request-response"
```

Comment:
👍 to adding the domain. Thanks for the links

I see it being a bit confusing but I'm having trouble thinking of an answer that _isn't_ confusing which i think is inherent to this message being a "response". I could change it to maybe `file-transfer-request-approval` but that might not indicate that it could be a denial (if I send a `Approved: false`) or maybe something silly like `file-transfer-request-answer` ? 

I'll move these to `api/constants` since I didn't find a more reasonable spot.